src/gui/src/components/analyze/ReportItem.vue

```diff
@@ -250,6 +259,44 @@ export default {
         })
     }
 
+    const cloneReportItem = async () => {
+      try {
+        console.log(
+          'Attempting to clone report item with state:',
+          report_item.value
+        )
+        // Prepare the data for cloning excluding fields we don't want duplicated
+        const cloneData = { ...report_item.value }
+        delete cloneData.id
+        delete cloneData.attributes
+        delete cloneData.created
+        delete cloneData.last_updated
+        delete cloneData.user_id
+        cloneData.uuid = null
+
+        console.log('Clone data before sending to backend:', cloneData)
+
+        const response = await createReportItem(cloneData)
+        notifySuccess(t('Clone created with ID:') + ` ${response.data.id}`)
+
+        const clonedItemId = response.data.id // Retrieve ID of the newly created item
+
+        // Prepare the update data with attributes
+        const updateData = {
+          attributes: report_item.value.attributes
+        }
+
+        // Update the newly created report item to include attributes
+        await updateReportItem(clonedItemId, updateData)
+        notifySuccess(`Attributes added to the clone with ID: ${clonedItemId}`)
+
+        router.push('/report/' + response.data.id)
+      } catch (error) {
+        console.error('Clone creation error:', error)
+        notifyFailure(t('Failed to clone report item'))
+      }
+    }
+
```

Comment:
When you clone a report that is not saved yet, an error is thrown (seems correct). However, if I go back to the Analyze view, a new empty entry is stored in the DB. Please, have a look if this is a bug or something intended.